save-core/src/commonTest/kotlin/org/cqfn/save/core/MergeConfigsTest.kt

```diff
@@ -141,10 +141,10 @@ class MergeConfigsTest {
         val config3 = TestConfig(toml3, config2, mutableListOf(generalConfig3, warnConfig3, fixConfig3))
         val config4 = TestConfig(toml4, config3, mutableListOf(generalConfig4, warnConfig4, fixConfig4))
 
-        config1.mergeConfigWithParent()
-        config2.mergeConfigWithParent()
-        config3.mergeConfigWithParent()
-        config4.mergeConfigWithParent()
+        config1.mergeConfigWithParents()
```

Comment:
I think we don't need merge each config with it parent. The main logic of such tests - call the merge function from some place in hierarchy tree (top, bottom, middle) and check, that all configs were successfully merged